files/usr/share/cinnamon/cinnamon-settings/modules/cs_themes.py

```diff
@@ -248,8 +248,8 @@ def refresh_chooser(self, payload):
 
                     dump = True
 
-                if theme_path == None:
-                    continue;
+                if theme_path is None:
+                    continue
```

Comment:
Your other PR for semicolons broke this one, can you rebase it?

Reply:
@mtwebster rebased. Should be fixed